app/controllers/issues_controller.rb

```diff
@@ -203,7 +203,9 @@ def update
     saved = false
     begin
       saved = save_issue_with_child_records
-    rescue ActiveRecord::StaleObjectError
+    rescue ActiveRecord::StaleObjectError => e
+      ZAppEngine::Api.log("Error::FailedToSaveIssue::#{@issue.id} with exception #{e}")
+      ZAppEngine::Api.log("Error::FailedToSaveIssue::#{@issue.id} with backtrace \n\t#{e.backtrace.join("\n\t")}")
```

Comment:
I would prefer using another method for logging since we have a dependency of redmine to a plugin by adding this here. Prefer rails logging instead.